torchft/futures_test.py

```diff
@@ -13,6 +14,8 @@
     stream_timeout,
 )
 
+_TIMEOUT_MANAGER._watchdog_interval = timedelta(seconds=1)
```

Comment:
I imagine this modifies the global state and could affect other tests? Will that cause any issues?

Reply:
That's a good catch -- we should move this into setUp/tearDown to reset it after